test/UnitTests/GitHub.App/ViewModels/GitHubPane/GitHubPaneViewModelTests.cs

```diff
@@ -23,9 +23,9 @@ public class TheInitializeMethod
         [Fact]
         public async Task NotAGitRepositoryShownWhenNoRepository()
         {
-            var te = Substitute.For<ITeamExplorerServiceHolder>();
-            te.ActiveRepo.Returns(default(ILocalRepositoryModel));
-            var target = CreateTarget(teServiceHolder: te);
+            var te = Substitute.For<ITeamExplorerContext>();
+            te.ActiveRepository.Returns(default(ILocalRepositoryModel));
+            var target = CreateTarget(teamExplorerContext: te);
```

Comment:
Now that we're using a properly mockable class for the team explorer stuff, we can probably simplify these tests...

Reply:
What did you have in mind?

Reply:
A lot of tests are creating a mock `ITeamExplorerContext` that returns an `ActiveRepository` of `null`. Could this could be made the default in `CreateTarget` if no mock is passed in?